src/DiagnosticMessages.ts

```diff
@@ -605,6 +605,11 @@ export let DiagnosticMessages = {
         message: `Missing exception expression after 'throw' keyword`,
         code: 1118,
         severity: DiagnosticSeverity.Error
+    }),
+    malformedTernaryExpression: () => ({
```

Comment:
Isn't there an unused code you could pick?

Reply:
Oh yeah, we had one left (1109). I switched to that.